include/api.php

```diff
@@ -2470,7 +2470,7 @@ function api_format_items($r,$user_info, $filter_user = false, $type = "json") {
 
 		$ret = Array();
 
-		foreach($r as $item) {
+		foreach ($r as $item) {
 
 			localize_item($item);
 			list($status_user, $owner_user) = api_item_get_user($a,$item);
```

Comment:
Found 41 occurrences in other files, fixing this one and marking it in todos.txt